src/Umbraco.Web/Security/AuthenticationExtensions.cs

```diff
@@ -169,18 +169,6 @@ public static bool RenewUmbracoAuthTicket(this HttpContextBase http)
             return true;
         }
 
-        /// <summary>
-        /// This will force ticket renewal in the OWIN pipeline
-        /// </summary>
-        /// <param name="http"></param>
-        /// <returns></returns>
-        internal static bool RenewUmbracoAuthTicket(this HttpContext http)
```

Comment:
I assume we don't use this anymore?

Reply:
It is still used in the  `AuthenticateCurrentRequest` method